modules/perception/inference/libtorch/BUILD

```diff
@@ -0,0 +1,19 @@
+load("//tools:cpplint.bzl", "cpplint")
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "libtorch_net",
+    srcs = ["libtorch_net.cc"],
+    hdrs = ["libtorch_net.h"],
+    deps = [
```

Comment:
I seems better if the files renamed to `torch_net.{h,cc}`, since the class in these files is `TorchNet`.